src/altair/block_processing.rs

```diff
@@ -55,9 +186,73 @@ pub fn process_deposit<
         MAX_VALIDATORS_PER_COMMITTEE,
         SYNC_COMMITTEE_SIZE,
     >,
-    _deposit: &mut Deposit,
-    _context: &Context,
+    deposit: &mut Deposit,
+    context: &Context,
 ) -> Result<()> {
+    let branch = deposit
+        .proof
+        .iter()
+        .map(|node| Node::from_bytes(node.as_ref().try_into().unwrap()))
+        .collect::<Vec<_>>();
+    let leaf = deposit.data.hash_tree_root()?;
+    let depth = crate::altair::DEPOSIT_CONTRACT_TREE_DEPTH + 1;
```

Comment:
```suggestion
    let depth = DEPOSIT_CONTRACT_TREE_DEPTH + 1;
```
ditto on imports of constants